v2/bigtable-cdc-to-hbase/src/main/java/com/google/cloud/teleport/v2/templates/BigtableCdcToHbase.java

```diff
@@ -0,0 +1,281 @@
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.beam.runners.dataflow.options.DataflowPipelineOptions;
+import org.apache.beam.sdk.Pipeline;
+import org.apache.beam.sdk.PipelineResult;
+import org.apache.beam.sdk.io.gcp.bigtable.BigtableIO;
+import org.apache.beam.sdk.io.hbase.HBaseIO;
+import org.apache.beam.sdk.options.Default;
+import org.apache.beam.sdk.options.ExperimentalOptions;
+import org.apache.beam.sdk.options.PipelineOptionsFactory;
+import org.apache.beam.sdk.values.KV;
+import org.apache.beam.sdk.values.PCollection;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.hbase.HBaseConfiguration;
+import org.apache.hadoop.hbase.client.RowMutations;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Bigtable change stream pipeline to replicate changes to Hbase. Pipeline reads from a Bigtable
+ * change stream, converts change stream mutations to their nearest Hbase counterparts, and writes
+ * the resulting Hbase row mutations to Hbase.
+ *
+ * <p>In Bigtable, all writes to a single row on a single cluster will be streamed in order by
+ * commitTimestamp. This is only the case for instances with a single cluster, or if all writes to a
+ * given row happen on only one cluster in a replicated instance. This order is maintained in the
+ * Dataflow connector via key-ordered delivery, as the Dataflow connector emits ChangeStreamMutation
+ * records with the row key as the record key.
+ */
+@Template(
+    name = "bigtable-cdc-to-hbase",
+    category = TemplateCategory.STREAMING,
+    displayName = "Bigtable CDC to HBase Replicator",
+    description = "A streaming pipeline that replicates Bigtable change stream data to HBase",
+    optionsClass = BigtableCdcToHbase.BigtableToHbasePipelineOptions.class,
+    flexContainerName = "bigtable-cdc-to-hbase",
+    contactInformation = "https://cloud.google.com/support")
+public class BigtableCdcToHbase {
+
+  private static final Logger LOG = LoggerFactory.getLogger(BigtableCdcToHbase.class);
+  private static final String USE_RUNNER_V2_EXPERIMENT = "use_runner_v2";
+
+  /** Options to run pipeline with. */
+  public interface BigtableToHbasePipelineOptions
+      extends DataflowPipelineOptions, ExperimentalOptions {
+    /** Bigtable change stream configs. */
+    @TemplateParameter.Text(
+        description = "Bigtable project id",
+        helpText = "GCP project id that contains Bigtable instance")
+    String getBigtableProjectId();
+
+    void setBigtableProjectId(String bigtableProjectId);
+
+    @TemplateParameter.Text(
+        description = "Bigtable instance",
+        helpText = "Bigtable instance to stream changes from")
+    String getInstanceId();
+
+    void setInstanceId(String instanceId);
+
+    @TemplateParameter.Text(
+        description = "Table id",
+        helpText = "The name of the Bigtable table whose changes to stream")
+    String getTableId();
+
+    void setTableId(String tableId);
```

Comment:
Can we try to consolidate / share options class between this and Bigtable CDC to BigQuery (https://github.com/GoogleCloudPlatform/DataflowTemplates/pull/628)? e.g.,  BigtableCdcReadOptions

Reply:
We would like to merge this in by EOM. Talked to Alexey, we can merge this first and I can add IT and rebase pipeline options from his template after his is merged in as well.

Reply:
Changed to extend bigtable-common options